.github/workflows/analyze-commits.yml

```diff
@@ -22,6 +22,23 @@ on:
         type: boolean
         default: true
         required: false
+      semanticReleasePlugins:
```

Comment:
Please, add new inputs to the readme

Reply:
@ilfa Updated. Could you please check the new changes in `README.md` file?